evalml/pipelines/components/estimators/regressors/multiseries_time_series_baseline_regressor.py

```diff
@@ -0,0 +1,112 @@
+"""Time series estimator that predicts using the naive forecasting approach."""
+import numpy as np
+import pandas as pd
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+from evalml.utils import infer_feature_types
+
+
+class MultiseriesTimeSeriesBaselineRegressor(Estimator):
+    """Multiseries time series regressor that predicts using the naive forecasting approach.
+
+    This is useful as a simple baseline estimator for multiseries time series problems.
+
+    Args:
+        gap (int): Gap between prediction date and target date and must be a positive integer. If gap is 0, target date will be shifted ahead by 1 time period. Defaults to 1.
+        forecast_horizon (int): Number of time steps the model is expected to predict.
+        random_seed (int): Seed for the random number generator. Defaults to 0.
+    """
+
+    name = "Multiseries Time Series Baseline Regressor"
+    hyperparameter_ranges = {}
+    """{}"""
+    model_family = ModelFamily.BASELINE
+    """ModelFamily.BASELINE"""
+    is_multiseries = True
+    supported_problem_types = [
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ]
+    """[
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ]"""
+
+    def __init__(self, gap=1, forecast_horizon=1, random_seed=0, **kwargs):
+        self._prediction_value = None
+        self.start_delay = forecast_horizon + gap
+        self._num_features = None
+
+        if gap < 0:
+            raise ValueError(
+                f"gap value must be a positive integer. {gap} was provided.",
+            )
+
+        parameters = {"gap": gap, "forecast_horizon": forecast_horizon}
+        parameters.update(kwargs)
+        super().__init__(
+            parameters=parameters,
+            component_obj=None,
+            random_seed=random_seed,
+        )
+
+    def fit(self, X, y=None):
+        """Fits multiseries time series baseline regressor to data.
+
+        Args:
+            X (pd.DataFrame): The input training data of shape [n_samples, n_features].
+            y (pd.Series): The target training data of length [n_samples].
```

Comment:
I think this needs to be updated to take in the data unstacked and `y` in a dataframe right?